extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -269,6 +269,7 @@ oppia.directive('graphViz', function() {
               y: $scope.state.mouseY,
               label: ''
             });
+            setMode0(_MODES.MOVE);
```

Comment:
setMode is callback for onClick (defined on $scope)
setModeHandler (private to controller, abstracted so that can be used multiple times, in setMode as well as onClickSVG callback) actually handles actual mode change